requirements-iree-pinned.txt

```diff
@@ -5,6 +5,4 @@
 --find-links https://iree.dev/pip-release-links.html
 iree-base-compiler==3.1.0rc20241204
 iree-base-runtime==3.1.0rc20241204
-
-# TODO(#760): include iree-turbine in this requirements file too?
-# iree-turbine==3.1.0rc20241205
+iree-turbine==3.1.0rc20241205
```

Comment:
Should the `iree-base-*` dependencies match with the `iree-turbine` version?

Reply:
I'll need to sync this. They are matching now: https://github.com/nod-ai/shark-ai/blob/d02d6e05d68c2d8f0dbda61d68c33c1418bff570/requirements-iree-pinned.txt#L3-L7

`3.1.0rc20241205` was the first nightly iree-turbine release but I didn't want to update the iree-base-* versions in this PR